<?xml version="1.0"?>

<launch>
    <param name="searching_altitude" value="1.2" /> <!-- in m -->
    <param name="takeoff_sleep_duration" value="3" /> <!-- in s -->
    <param name="takeoff_vel_z" value="0.8" /> <!-- in m/s -->


    <node name="platform_tracking_real" pkg="vision" type="platform_tracking_real" output="screen">
        <param name="Kp" value="0.04"/>
    	<param name="Ki" value="0.0"/>
    	<param name="Kd" value="0.01"/>

        <!--TOPICS-->
        <param name="cmd_topic_vel" value="/cmd_vel"/>
        <param name="platform_position_topic" value="/aruco_single/pose"/>
        <param name="ardrone_navdata" value="/ardrone/navdata"/>
        
   		<param name="takeoff_topic" value="/ardrone/takeoff" />
        <param name="landing_topic" value="/ardrone/land" />
        <param name="emergency_topic" value="/ardrone/emergency_topic" />
        <param name="tracking_topic" value="/ardrone/tracking_topic" />
    </node>
</launch>
